### PR TITLE
Codemod `fix-assert-tuple`

### PR DESCRIPTION
**Pull request by clavedeluna**
## Overview
Codemod to convert `assert (1, 2)` into 
```
assert 1
assert 2
```

Notice the same sonarcloud rule this PR triggered and [this is](https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/assert-on-tuple.html) the same pylint rule

### CODE REVIEW

File: src/core_codemods/fix_assert_tuple.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        ):
+            return updated_node
+
+        if len(updated_node.body) == 1 and isinstance(
+            assert_node := updated_node.body[0], cst.Assert
+        ):
+            match assert_test := assert_node.test:
+                case cst.Tuple():
+                    if not assert_test.elements:
+                        return updated_node
```
Comment (andrecsilva) [inline src/core_codemods/fix_assert_tuple.py:33]: Suggestion:
```python
match updated_node.body:
    case [cst.Assert(test=cst.Tuple(elements = [head, *_]))]:
        ...
```

File: src/core_codemods/docs/pixee_python_fix-assert-tuple.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+An assertion on a non-empty tuple will always evaluate to `True` so it's likely to be written unintentionally. This codemod will write one assert statement for each item in the tuple.
+
+
+
```
Comment (andrecsilva) [inline src/core_codemods/docs/pixee_python_fix-assert-tuple.md]: I feel those empty lines shouldn't be here.

**Review by andrecsilva — APPROVED** [review]

File: src/core_codemods/fix_assert_tuple.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+from typing import List, Union
+from core_codemods.api import Metadata, ReviewGuidance, SimpleCodemod
+from codemodder.change import Change
+from codemodder.codemods.utils_mixin import NameResolutionMixin
+
+
+class FixAssertTuple(SimpleCodemod, NameResolutionMixin):
+    metadata = Metadata(
+        name="fix-assert-tuple",
+        summary="Fix Assert on Populated Tuple",
```
Comment (drdavella) [inline src/core_codemods/fix_assert_tuple.py]: Suggestion:
```suggestion
        summary="Fix `assert` on Non-Empty Tuple Literal",
```

File: src/core_codemods/fix_assert_tuple.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        review_guidance=ReviewGuidance.MERGE_WITHOUT_REVIEW,
+        references=[],
+    )
+    change_description = (
+        "Separate assertion on a populated tuple into multiple assert statements."
```
Comment (drdavella) [inline src/core_codemods/fix_assert_tuple.py]:
```suggestion
        "Separate assertion on a non-empty tuple literal into multiple assert statements."
```

File: src/core_codemods/docs/pixee_python_fix-assert-tuple.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+An assertion on a non-empty tuple will always evaluate to `True` so it's likely to be written unintentionally. This codemod will write one assert statement for each item in the tuple.
```
Comment (drdavella) [inline src/core_codemods/docs/pixee_python_fix-assert-tuple.md]:
```suggestion
An assertion on a non-empty tuple will always evaluate to `True`. This means that `assert` statements involving non-empty tuple literals are likely unintentional and should be rewritten. This codemod rewrites the original `assert` statement by creating a new `assert` for each item in the original tuple.
```

File: src/core_codemods/fix_assert_tuple.py [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+from core_codemods.api import Metadata, ReviewGuidance, SimpleCodemod
+from codemodder.change import Change
+from codemodder.codemods.utils_mixin import NameResolutionMixin
+
+
+class FixAssertTuple(SimpleCodemod, NameResolutionMixin):
+    metadata = Metadata(
+        name="fix-assert-tuple",
+        summary="Fix Assert on Populated Tuple",
+        review_guidance=ReviewGuidance.MERGE_WITHOUT_REVIEW,
```
Comment (drdavella) [inline src/core_codemods/fix_assert_tuple.py]: I think this should be `MERGE_AFTER_CURSORY_REVIEW` because there's a decent change we break some tests.

**Review by drdavella — CHANGES_REQUESTED** [review]
Very nice overall but some suggestions for metadata

File: src/core_codemods/fix_assert_tuple.py (hunk 1 above)
Comment (clavedeluna) [reply src/core_codemods/fix_assert_tuple.py:33]: I personally don't prefer this version

**Review by drdavella — APPROVED** [review]

**Pull request merged**